inception-ui-kb/src/main/java/de/tudarmstadt/ukp/inception/ui/kb/ConceptInstancePanel.java

```diff
@@ -163,17 +163,6 @@ public void actionStatementChanged(AjaxStatementChangedEvent event)
 //     * @return true if the event is a renaming event, false otherwise
 //     */
 //    private boolean isRenamingEvent(AjaxStatementChangedEvent aEvent)
-//    {
```

Comment:
Why not remove the commented code above?

Reply:
We did not remove these commented lines because these are describing the code above from line 129 to 151